test/any.specs.ts

```diff
@@ -0,0 +1,203 @@
+import "mocha"
+import fc, { Value } from "fast-check"
+import { areEquals, isEmpty, isNotEmpty } from "../src/any"
+import { strict as assert } from "assert"
+
+//isEmpty tests
+describe("isEmpty: ", () => {
+  it("Should return true for empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ))
+
+  it("Should return false for non-empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ minLength: 1 }), (value) => !isEmpty(value)
+      )
+    ))
+
+  it("Should return true for empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ));
+
+  it("Should return false for non-empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { minLength: 1 }), (value) => {
+        return !isEmpty(value)
+      })
+    ))
+
+  it("Should return true for empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 0 }), (value) => {
+        isEmpty(value)
+      })
+    ))
+
+  it("Should return false for non-empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 200 }), (value) => {
+        !isEmpty(value)
+      })
+    ))
+
+    it("Should return true for null or undefined values", () => {
+      fc.assert(fc.property(fc.constant(null), (value) => isEmpty(value)));
+      fc.assert(fc.property(fc.constant(undefined), (value) => isEmpty(value)));
+    });
+
+  it("Should return true for false boolean", () => {
+    fc.assert(fc.property(fc.constant(false), (boo) => isEmpty(boo) === true))
+  })  
+
+  it("Should return false for true boolean", () =>
+  fc.assert(
+    fc.property(fc.constant(true), (boo) => !isEmpty(boo))
+  )
+);
+
+  it("Should return true for BigInt or Number values equal to 0", () =>
```

Comment:
Add a constraint to the bigInt and integer instead of using a weird filter and condition, that will have better perf and is more readable.